tests/org.eclipse.n4js.xpect.tests/model/typemodifier/GHOLD411_optional_field_ternay_expression_object_literals.n4js.xt

```diff
@@ -23,8 +23,8 @@ var jvar: J = { };
 const jconst: J = {};
 var b: boolean;
 
-// XPECT warnings --> "The type of this expression is undefined, so it will never evaluate to a value other than undefined." at "b ? null : null" 
-i = b ? null : null;
```

Comment:
The test was wrong and only worked because before `union{null, null}` was (incorrectly) "simplified" to `undefined`, which triggered the warning "The type of this expression is undefined, so ...". After fixing the simplification, I had to change `null` to `undefined` to obtain the same behavior.